From 41322f54b7d8be24da30cd58d1b085f608da51ed Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Thu, 7 May 2026 13:51:57 +0800
Subject: [PATCH] configs: evb-ast2705: enable ASPEED XGMAC

Enable the ASPEED DWC XGMAC glue driver in the AST2705 EVB
defconfig so the board builds in 10G Ethernet support.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: I6a2b7a375a2f31b357c3258eac529bd34598aac2
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 configs/evb-ast2705_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/evb-ast2705_defconfig b/configs/evb-ast2705_defconfig
index 1954e04b12..7e343c82bc 100644
--- a/configs/evb-ast2705_defconfig
+++ b/configs/evb-ast2705_defconfig
@@ -139,6 +139,7 @@ CONFIG_PHY_NCSI=y
 CONFIG_PHY_AIROHA=y
 CONFIG_PHY_AIROHA_AN8801=y
 CONFIG_DM_MDIO=y
+CONFIG_DWC_ETH_XGMAC_ASPEED=y
 CONFIG_FTGMAC100=y
 CONFIG_ASPEED_MDIO=y
 CONFIG_DM_PCI_COMPAT=y
-- 
2.43.0

